.sqlfluff

```diff
@@ -1,18 +1,21 @@
 [sqlfluff]
+templater = jinja
```

Comment:
what's the motivation for this change?

Reply:
long story, but i'm going to cut it back to dbt-templater i just wanted to get the PRs reconciled -- doing that now